lib/application_json.rb

```diff
@@ -113,6 +113,14 @@ def applications_params
         type: 'string',
         description: 'Optional. Include only applications changed or created on
         or since a date and time. Times should be in ISO 8601 format.'
+      },
+      {
+        name: 'order',
+        type: 'string',
+        description: 'Optional. "asc" or "desc". Specify how the returned
```

Comment:
Would it make sense to make this explicit? Eg `?order=newest_first` and `?order=oldest_first`?

Reply:
Much better. Will do

Reply:
Often `sort_by` is used to name this param. 
For example `?sort_by=email`

Reply:
Making it generic comes at a small cost of explicitness, and because it's an additive change we can always do it later. For these reasons I'd prefer to stick to oldest_first and newest_first for now.